### PR TITLE
SDIT-2319 Add suggestions placeholder

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: server/routes/search/searchController.ts [hunk 1]
```diff
@@ -66,7 +66,6 @@ export default class SearchController {
     const pagedResults: PagedModelPrisonerSearchDto = await this.doSearch(req, res)
 
     const paginationParams = this.getPaginationParams(req, pagedResults.page)
-    logger.debug('paginationParams', paginationParams)
```
Comment (karenmillermoj) [inline server/routes/search/searchController.ts:69]: Thought I still needed it, but I don't - picked up in previous PR

File: server/routes/search/searchController.ts [hunk 2]
```diff
@@ -101,6 +100,12 @@ export default class SearchController {
     }
   }
 
+  getSuggestions(req: Request, res: Response) {
+    logger.debug('GET /search')
+    // TODO
```
Comment (karenmillermoj) [inline server/routes/search/searchController.ts:105]: Not just rendering here - need to add information to display in the page

File: server/views/pages/search.njk [hunk 3]
```diff
@@ -33,49 +33,55 @@
...
-                        Reception {{ prisoner.receptionDate | formatDate }}
-                      </div>
-                      {% if prisoner.isAlias %}
-                        <div class="govuk-body govuk-!-margin-bottom-1" data-qa="alias">
-                          Matched on alias {{ prisoner.aliasFirst }} {{ prisoner.aliasLast }}
+                  {% set rows = [] %}
+                  {% for prisoner in searchResults %}
+                    {% set cellHtml %}
+                      <div class="govuk-grid-column-three-quarters">
+                        <div class="govuk-body-lead govuk-!-font-weight-bold govuk-!-margin-bottom-1">
```
Comment (mikehalmamoj) [inline server/views/pages/search.njk:46]: Feels like a macro might be good here

**Review by mikehalmamoj — APPROVED** [review]

File: server/views/pages/search.njk (hunk 3 above)
Comment (karenmillermoj) [reply server/views/pages/search.njk:46]: good idea - just been looking at those actually

**Pull request merged**